analyser/panako_processor.py

```
@@ -3,417 +3,69 @@
-        raise RuntimeError(error_msg)
-    except Exception as e:
-        error_msg = f"Error extracting fingerprints: {e}"
-        print(error_msg)
-        print(traceback.format_exc())
-        raise
 
 
-def get_next_previews_to_fingerprint(id_token, batch_size=10):
+def get_next_previews_to_fingerprint(batch_size=10):
     print("Getting next previews to fingerprint")
     res = requests.get(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)